### PR TITLE
Update `exec` command for newer version of Yargs

### PR DESCRIPTION
**Pull request by taymoork2**
Fixes [SPARK-207457](https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-207457)

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by mwegman — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "srcdist": "sed -ibak 's#\"main\": \"src/index.js#\"main\": \"dist/index.js#' packages/node_modules/{*,@webex/*}/package.json && rimraf packages/node_modules/{*,@webex/*}/*bak",
     "get-current-version": "node ./tooling/index.js version current",
     "get-next-version": "node ./tooling/index.js version next --alwaysIncrement --includeSamples",
-    "deps:check": "node ./tooling/index.js exec -- dependency-check ./package.json",
+    "deps:check": "node ./tooling/index.js exec --cmd dependency-check -- ./package.json",
```
Comment (adamweeks) [inline package.json:46]: This command is failing when I test on this branch. Where it is even being used?

File: .circleci/config.yml [hunk 2]
```diff
@@ -366,7 +366,7 @@ jobs:
               npm run deps:generate
 
               echo "Publish to NPM"
-              npm run tooling -- exec -- bash -c 'npm publish --access public || true'
+              npm run tooling -- exec --cmd bash -- -c 'npm publish --access public || true'
```
Comment (adamweeks) [inline .circleci/config.yml:369]: Tested with `npm run tooling -- exec --cmd bash -- -c 'ls -l'  ` and it behaved properly!

**Review by adamweeks — APPROVED** [review]
Approved for hotfix. Not sure about the `deps:check` command.

File: package.json (hunk 1 above)
Comment (taymoork2) [reply package.json:46]: I tried this on a branch that didnt have the yargs changes on it, it was still failing ¯\_(ツ)_/¯

**Pull request merged**